### ISSUE TITLE
New feature request: command line user creation

### ISSUE THREAD

**Issue by mesmoiron** [decidim/metadecidim#126]
Hello,

I am testing Decidim locally. When you use the seed data everything works fine. However when you do a manual clean install, it is impossible to access the admin page. Admin is a usual user. That is, the localhost:3000/admin/ routes to the /user_login page. It is impossible to use the new admin created in the localhost:3000/system/ dashboard. Since, the application relies on e-mail verification it won't work.

To recap:
- a terminal function to create users and admins
- the possibility to disable e-mail verification in the system admin dashboard for an admin
-  the system admin should be able to also login to the /admin dashboard: these functions should and could be the same for some installs.
